schema.json

```diff
@@ -1344,6 +1375,183 @@
         "type": "color",
         "label": "Text hover color",
         "id": "card--alternate-color--hover"
+      },
+      {
+        "type": "heading",
+        "content": "Image sizes"
+      },
+      {
+        "type": "paragraph",
+        "content": "Please specify a maximum image size for desktop display. We'll automatically scale down the image for smaller devices.",
+        "hideExpression": "model.productSize != 'product'"
+      },
+      {
+        "type": "select",
+        "label": "Main product images",
+        "id": "productSize",
+        "force_reload": true,
+        "options": [
+          {
+            "value": "500x659",
+            "label": "Optimized for theme"
+          },
+          {
+            "value": "product",
+            "label": "Specify dimensions"
+          }
+        ]
+      },
+      {
+        "type": "number",
+        "id": "_images.product.width",
+        "force_reload": true,
+        "label": "Max width",
+        "className": "formly-field-dimensions",

```

Comment:
I'm not sure why designers need to now what class names are used in theme editor. I think we should remove this attribute and use one style in theme-editor